articles/sso/v2/single-page-apps-sso.md

```diff
@@ -0,0 +1,134 @@
+---
+description: How to implement client-side SSO on single page applications.
+toc: true
+---
+
+# Client-Side SSO on Single Page Applications
+
+:::panel-warning Prerequisites
+This article assumes that you have already created an Auth0 Client (with its type set to **Single Page Web Applications**) with the **OIDC Conformant** flag enabled.
+:::
+
+To bypass displaying the Lock screen when logging in a user (a process known as [*silent authentication*](#silent-authentication)), you must:
+
+* Enable the **Use Auth0 instead of the IdP to do Single Sign On** flag in the [Auth0 Client's settings tab](${manage_url});
```

Comment:
I think it's better to replace the URL with sth like `${manage_url}/#/clients/${client.accountId}/settings`.